src/bin/jpwl/convert.c

```diff
@@ -135,15 +131,19 @@ static int tga_readheader(FILE *fp, unsigned int *bits_per_pixel,
     id_len = (unsigned char)tga[0];
     cmap_type = (unsigned char)tga[1];
     image_type = (unsigned char)tga[2];
-    cmap_index = get_ushort(*(unsigned short*)(&tga[3]));
+    cmap_index = get_tga_ushort(*(unsigned short*)(&tga[3]));
     cmap_len = get_ushort(*(unsigned short*)(&tga[5]));
+    cmap_len = get_tga_ushort(*(unsigned short*)(&tga[5]));
```

Comment:
I suspect this doesn't compile... Line 135 should be removed as it is effectively overrideen by line 136. Similar instances below

Reply:
I have fixed that by make a new pr.